src/main/java/macrobase/analysis/stats/mixture/BatchMixtureModel.java

```diff
@@ -91,7 +100,7 @@ public double getZScoreEquivalent(double zscore) {
     }
 
     public boolean checkTermination(double logLikelihood, double oldLogLikelihood, int iteration) {
-        log.debug("log likelihood after iteration {} is {}", iteration, logLikelihood);
+        log.info("average point log likelihood after iteration {} is {}", iteration, logLikelihood);
```

Comment:
debug doesn't show up when I'm running diagnostic tasks. I didn't know how to set a not default log level there.